File: server/java-api/myproject/src/main/java/com/javaapi/app/user/core/dto/SessionDTO.java
package com.javaapi.app.user.core.dto;


public class SessionDTO {
    private  String userId;
    private  String username;
    private  String email;



    public SessionDTO(String userId, String username, String email) {
        this.userId = userId;
        this.username = username;
        this.email = email;
    }
    public SessionDTO() {}


    public String getUserId() {
        return userId;
    }

    public String getUsername() {
        return username;
    }

    public String getEmail() {
        return email;
    }

    public String setUserid(String userId) {
        return userId;
    }
    public String setUsername(String username) {
        return username;
    }
    public String setEmail(String email) {
        return email;
    }
}

File: server/go-ws/internal/app/core/dto/session/SessionDTO.go
package session

type SessionDTO struct {
	UserID   string 
	Username string 
	Email    string 
}

func NewSessionDTO(userID, username, email string) *SessionDTO {
	return &SessionDTO{
		UserID:   userID,
		Username: username,
		Email:    email,
	}
}

func (dto *SessionDTO) GetUserID() string {
	return dto.UserID
}	

func (dto *SessionDTO) GetUsername() string {
	return dto.Username
}

func (dto *SessionDTO) GetEmail() string {
	return dto.Email
}	